docs/reference/fields.txt

```diff
@@ -1386,8 +1386,39 @@ requiring the usage of the aggregation pipeline for both queries and updates,
 applications should avoid using dots in field names and starting field names
 with the dollar sign if possible.
 
-The Ruby driver `currently prohibits
-<https://jira.mongodb.org/browse/RUBY-2528>`_ inserting documents whose
-field names contain dots or begin with the dollar sign. However, if such
-documents are inserted using other software, Mongoid and the Ruby driver
-provide limited support for retrieving and operating on these documents.
+Mongoid, starting in version 8, now allows users to access fields that begin with
```

Comment:
This works in previous Mongoid versions also, doesn't it? If so I would remove the "starting in version 8" part.

Reply:
Not exactly... Technically accessing fields that start with a `$` would work, but accessing dotted fields would not.